__init__.py

```diff
@@ -44,8 +45,6 @@
             print("The Retriever lacks permission to access the ~/.retriever/ directory.")
             raise
 SCRIPT_SEARCH_PATHS = [
```

Comment:
These extra paths are by design. The idea is that if you want to add a custom script then you can place it in your working directory or in a `scripts` subfolder. The `scripts` folder is also crucial for testing new scripts because it lets the script in the repo get tested without moving it manually into `~/.retriever/scripts`.

@goelakash - was there a reason you needed to take these out?